libethash-cl/ethash_cl_miner.h

```diff
@@ -65,6 +65,14 @@ class ethash_cl_miner
 	void hash_chunk(uint8_t* _ret, uint8_t const* _header, uint64_t _nonce, unsigned _count);
 	void search_chunk(uint8_t const*_header, uint64_t _target, search_hook& _hook);
 
+	/* -- default values -- */
+	/// Default value of the local work size. Also known as workgroup size.
+	static const unsigned defaultLocalWorkSize;
```

Comment:
coding standards: should either be `c_`... if not public API or CamelCase if yes.